py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -203,6 +203,17 @@ def test_from_pydatetime() -> None:
     assert s.dt[0] == dates[0]
 
 
+def test_from_numpy_timedelta() -> None:
+    s = pl.Series(
+        "name",
+        np.array([timedelta(days=1), timedelta(seconds=1)], dtype="timedelta64[ms]"),
+    )
+    assert s.dtype == pl.Duration
```

Comment:
could we check the time_unit of the result as well please?

could be a good use case for parametrisation, something like
```python
@pytest.mark.parametrize('time_unit', ['ms', 'us', 'ns'])
def test_from_numpy_timedelta(time_unit) -> None:
    s = pl.Series(
        "name",
        np.array([timedelta(days=1), timedelta(seconds=1)], dtype=f"timedelta64[{time_unit}]"),
    )
    assert s.dtype == pl.Duration(time_unit)
```

Reply:
Sure!